electricitymaps/electricitymaps-contrib
# feat(web): US Specific survey

## Issue
We want to understand our US user more so we can build great features 🦅

AVO-425

## Description

This PR adds a new in app survey that targets US specific users. The targeting is currently taken from the userLocation on map load and will not be displayed if users load directly to a zone. 

We could consider adding a trigger when any US zone is clicked that would widen the funnel of users we're surveying. 

## Testing instructions
Add the `ff` query param i.e. `app.electricitymaps.com/map?ff` to open the feature flags manager and toggle on the survey then you'll also need a VPN that locations you inside the US. The survey should appear on map load when not viewing a zone. If testing is painful add a comment and I'll make it easier 👍
